### PR TITLE
Adding a new structure for the sass files

### PR DESCRIPTION
**Pull request by znegrin**
After a conversation with @dpordomingo and referred to the task #83 I added a new division of folders and individual files for `abstracts`, `base`, `components`, `layout` and `pages` to make it more scalable and organized.

### CODE REVIEW

File: src/sass/base/_typography.scss [hunk 1]
```diff
@@ -1,16 +1,20 @@
-@import "~@fortawesome/fontawesome-free/scss/fontawesome.scss";
-@import "~@fortawesome/fontawesome-free/scss/solid.scss";
-@import "~@fortawesome/fontawesome-free/scss/regular.scss";
+/**
+ * Typography definitions
+ */
+
+@import "../../../node_modules/@fortawesome/fontawesome-free/scss/fontawesome";
```
[the remark this answers is not in the document]
Comment (znegrin) [reply src/sass/base/_typography.scss]: I didn't change that, so I don't know. I just kept it as it was before.

**Comment by znegrin** [comment]
I did all the requested changes and suggestions.

File: src/sass/base/_typography.scss [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+@import "../../../node_modules/@fortawesome/fontawesome-free/scss/solid";
+@import "../../../node_modules/@fortawesome/fontawesome-free/scss/regular";
 
 @font-face {
     font-family: 'Cerebri';
-    src: url(../fonts/cerebrisans-book.woff);
+    src: url(../../fonts/cerebrisans-book.woff);
```
[the remark this answers is not in the document]
Comment (znegrin) [reply src/sass/base/_typography.scss]: I actually had to revert this as the abs path wasn't working.

File: src/sass/base/_typography.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (dpordomingo) [reply src/sass/base/_typography.scss]: But you can see it in the diff.
It was previously using this other format:
```suggestion
@import "~@fortawesome/fontawesome-free/scss/fontawesome.scss";
```

File: src/sass/base/_typography.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (dpordomingo) [reply src/sass/base/_typography.scss]: how come!
I just tried it locally and worked like a charm!
What error were you obtaining?

[the remark this answers is not in the document]
Comment (znegrin) [reply src/sass/base/_typography.scss]: It is not loading the font if I use your suggested change: 
`src: url('/src/fonts/cerebrisans-book.woff');`

image

**Pull request merged**